jamo/README.md

```diff
@@ -0,0 +1,10 @@
+# Running JAMO at NERSC
+
+This directory contains all the necessary files for setting up JAMO at NERSC. Running JAMO at NERSC
+is broken down into four parts:
+
+- Building the JAMO application as a Docker image. The relevant files for this can be found in `jamo/docker`.
+- Instantiating JAMO on NERSC's Spin infrastructure. The necessary files for this can be found in `jamo/k8s` and `jamo/config`.
+- Running the data transfer service on Perlmutter and the Data transfer nodes. The necessary files for this can be found in `jamo/dt_service`.
+
+See the wiki for more details on [setting up JAMO](https://github.com/lbl-cbg/t5-data/wiki/Setting-up-JAMO).
```

Comment:
Thanks for creating these setup instructions! I'll try to give them a spin in December to see if I could set JAMO up following these instructions.

It would be nice to also add docs on how to modify JAMO, e.g., "How to add a new dataset type".

Reply:
Sure, I'll add another page to the Wiki about this.

Reply:
Thanks!